yew-macro/tests/derive_variants/pass.rs

```diff
@@ -0,0 +1,199 @@
+#![recursion_limit = "128"]
+
+use yew::prelude::*;
+
+use yew::html::ChildrenRenderer;
+use yew::virtual_dom::{VChild, VComp};
+
+mod t1 {
```

Comment:
Instead of using modules, consider using multiple files. When changing the tests it often leads to line changes which creates a lot of noise in the `stderr` diff.
By splitting it up into multiple files we can hopefully reduce this.